pulpcore/pulpcore/app/urls.py

```diff
@@ -117,12 +118,29 @@ def __repr__(self):
     url(r'^api/v3/orphans/', OrphansView.as_view()),
 ]
 
-# if drf_openapi is installed add live docs route
+# if drf_yasg is installed add live docs route
 with suppress(ImportError):
```

Comment:
Since you added 'drf-yasg' to the requirements in setup.py we don't need to treat it as an optional dependency anymore, and can remove the suppress and move the import statements to the top.